citestwheel.Dockerfile

```
@@ -19,14 +19,24 @@ ARG DEBIAN_FRONTEND=noninteractive
 ENV PYENV_ROOT="/pyenv"
 ENV PATH="/pyenv/bin:/pyenv/shims:$PATH"
 
-RUN apt-get update \
-        && apt-get upgrade -y \
-        && apt-get install -y --no-install-recommends \
-        wget curl git jq ssh \
-        make build-essential libssl-dev zlib1g-dev \
-        libbz2-dev libreadline-dev libsqlite3-dev wget \
-        curl llvm libncursesw5-dev xz-utils tk-dev unzip \
-        libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
+RUN <<EOF
+set -e
+echo 'APT::Update::Error-Mode "any";' > /etc/apt/apt.conf.d/warnings-as-errors
+apt-get update
+apt-get install -y software-properties-common
+# update git > 2.17
+add-apt-repository ppa:git-core/ppa -y
+apt-get update
+apt-get upgrade -y
+apt-get install -y --no-install-recommends \
+  wget curl git jq ssh \
+  make build-essential libssl-dev zlib1g-dev \
+  libbz2-dev libreadline-dev libsqlite3-dev wget \
+  curl llvm libncursesw5-dev xz-utils tk-dev unzip \
+  libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev \
+  software-properties-common
```

We already have `software-properties-common` from above.
```suggestion
  libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
```